Localization/cubature_kalman_filter/cubature_kalman_filter.py

```diff
@@ -0,0 +1,234 @@
+"""
+implementation of cubature kalman filter using CTRV motion model
+
+# author: Raghuram Shankar
+# based on PythonRobotics by Atsushi Sakai
```

Comment:
I think this line is not needed.

Reply:
Okay sure, I will erase them.

Reply:
No. I mean only this line "based on PythonRobotics by Atsushi Sakai" is not needed. Other comments should be kept.

Reply:
Ah, okay. Got it.